src/Clay/Property.hs

```diff
@@ -106,6 +97,17 @@ intercalate s (x:xs) = foldl (\a b -> a `mappend` s `mappend` b) x xs
 
 -------------------------------------------------------------------------------
 
+newtype Number = Number { unNumber :: Fixed E5 }
```

Comment:
Can you add some haddocks here?

Reply:
I added this:
```haskell
-- | A number with fixed precision, supporting up to 5 decimal places.
```
Does this work? I'm not entirely sure what to put 😅

Reply:
Great :) I added a little suggestion.